Apps/RaptureIntTests/src/test/java/rapture/helper/IntegrationTestHelper.java

```diff
@@ -198,8 +198,8 @@ public void configureTestRepo(RaptureURI repo, String storage, boolean versioned
             break;
     
         case DOCUMENT:
-            if (docApi.docRepoExists(repo.toAuthString())) docApi.deleteDocRepo(authString);
-            	docApi.createDocRepo(authString, "NREP {} USING " + storage + " {prefix=\"D_" + repo.getAuthority()+ (versioned? ", separateVersion=\"true\"}":"") + "\"}");
+            if (docApi.docRepoExists(repo.toAuthString())) docApi.deleteDocRepo(authString);            
+            	docApi.createDocRepo(authString, "NREP {} USING " + storage + " {prefix=\"D_" + repo.getAuthority()+ (versioned? "\", separateVersion=\"true":"") + "\"}");
             Assert.assertTrue(docApi.docRepoExists(authString), authString + " Create failed");
```

Comment:
That looks OK but it's still hard to follow - would be easier to understand as 

docApi.createDocRepo(authString, "NREP {} USING " + storage + " {prefix=\"D_"+repo.getAuthority()+"\"" +(versioned ? ", separateVersion=\"true\" : "") + "}");